packages/accordions/src/styled/accordion/StyledRotateIcon.ts

```diff
@@ -22,12 +22,17 @@ interface IStyledRotateIcon {
   $isCollapsible?: boolean;
 }
 
-const colorStyles = (props: ThemeProps<DefaultTheme> & any) => {
-  const showColor = props.$isCollapsible || !props.$isRotated;
-  let color = getColorV8('neutralHue', 600, props.theme);
+const colorStyles = ({
+  $isCollapsible,
+  $isHovered,
+  $isRotated,
+  theme
+}: ThemeProps<DefaultTheme> & any) => {
+  const showColor = $isCollapsible || !$isRotated;
+  let color = getColor({ theme, variable: 'foreground.subtle' });
 
-  if (showColor && props.$isHovered) {
-    color = getColorV8('primaryHue', 600, props.theme);
+  if (showColor && $isHovered) {
+    color = getColor({ theme, variable: 'foreground.primary' });
   }
```

Comment:
```suggestion
  if (showColor && $isHovered) {
    color = getColor({ theme, variable: 'foreground.primary' });
  } else {
    color = getColor({ theme, variable: 'foreground.subtle' });
  }
```

Would be slightly more efficient to just set `let color;` above. Either that, or use `const` with the `variable` set in a ternary.